card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -43,7 +45,7 @@ def default_modal_content_args args
   end
 
   view :modal_title do |args|
-    _render_title args.merge(:title_class=>'modal-title')
+    "<h4>#{_render_title args.merge(:title_class=>'modal-title')}</h4>"
```

Comment:
oh, then why do we need a modal_title view?  can't it just render the normal title with the args?

I was actually asking because conceptually it seemed like the card-header context would give you whatever you needed instead of using the h4, but maybe the h4 brings in more bootstrappiness?